Civi/Api4/Generic/Traits/CustomValueActionTrait.php

```diff
@@ -51,7 +51,7 @@ public function getEntityName() {
   /**
    * @inheritDoc
    */
-  protected function writeObjects($items) {
+  protected function writeObjects(&$items) {
     $fields = $this->entityFields();
     foreach ($items as $idx => $item) {
```

Comment:
Should `$items as $idx => $item` be `$items as $idx => &$item`?

Reply:
Actually no I think that would mess up the output values which uses the original `$items` not the rewritten keys.

Reply:
Ok